config/install/field.field.node.localgov_campaigns_page.localgov_campaigns_content.yml

```diff
@@ -92,21 +94,24 @@ settings:
       localgov_link_and_summary:
         enabled: true
         weight: -25
-      localgov_quote:
+    localgov_media_with_text:
```

Comment:
Looks to be an indentation error with this line.

Reply:
Fixed